app/controllers/posts_controller.rb

```diff
@@ -5,7 +5,7 @@ class PostsController < ApplicationController
   def index; end
 
   def show
```

Comment:
- [ ] Please make sure that the N+1 problem is solved when fetching all posts and their comments for a user as requested in the project requirements,
  - You can consider the code `@posts = @user.posts.includes(:comments)`.

Reply:
✔️ Fixed
Before:
![image](https://github.com/luis-pomare/rails-blog-app/assets/80157011/993ea0aa-4d60-420f-803a-0e34f86cdfe6)
After:
![image](https://github.com/luis-pomare/rails-blog-app/assets/80157011/4809d268-9fea-4343-b193-1fa94e49240e)